python/tchannel/statsd.py

```diff
@@ -38,6 +39,7 @@ def __init__(self, statsd):
         :param statsd: instance of `StatsD <https://github.com/etsy/statsd>`
         """
         self._statsd = statsd
+        self.outbound_attempt = {}
```

Comment:
Usually for dictionaries like this, it's a good idea to add a comment indicating what it will map from/to.